lib/efsm.ml

```diff
@@ -178,9 +187,41 @@ let powerset_construction (start, old_g) =
         Map.fold out_edges ~init:(g, state_map) ~f
   in
   let start_states = Map.find_exn epsilons start in
+  Stdio.print_endline
```

Comment:
Please remove this print.

Reply:
Sure, I was planning on undoing all the changes I made and keeping the latest version on develop. In the end I didn't use any of the changes I made to the efsm